.travis.yml

```diff
@@ -38,11 +38,12 @@ script:
     cd ${CPU}/${buildOS}/${usage}/${imageName}
   - make
   - travis_retry ctest -V
-after_success:
-  - docker login -u ovckey -p $REGISTRY_PASS
-  - docker push $DOCKER_PREFIX/xeon-$(echo $buildOS | tr -d - | tr -d .)-$usage-$imageName:build 
-  - docker push $DOCKER_PREFIX/xeon-$(echo $buildOS | tr -d - | tr -d .)-$usage-$imageName:latest 
-
+after_success: |
+   if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
```

Comment:
Suggest not using double equal since its not guaranteed with single bracket test or just use double brackets since these run in bash

Reply:
Doesn't really matter either way